addons/web/static/src/js/components/pager.js

```diff
@@ -142,16 +135,18 @@ odoo.define('web.Pager', function (require) {
 
         /**
          * @private
-         * @param {Object} payload
+         * @param {number} currentMinimum
+         * @param {number} limit
          */
         _updateAndDisable(currentMinimum, limit) {
             if (
                 currentMinimum !== this.props.currentMinimum ||
                 limit !== this.props.limit
             ) {
                 this.state.disabled = true;
```

Comment:
couldn't you always disable it? That way, it would behave exactly the same whether the value changed or not.

Reply:
Indeed, done